pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -3325,6 +3341,132 @@ derived_table:
         $$ = &tree.ParenTableExpr{Expr: $2}
     }
 
+table_function:
```

Comment:
we will add more user defined table function support.  like:
https://github.com/matrixorigin/matrixone/issues/3840
so "unnest" maybe an builtin table function.  i don't known if make unnest be a keyword will be a good choice or let's make all string to be a table function name will better.

but I think that's ok now.  thanks your great work.    maybe you can challenge #3840 next

Reply:
Ok,I see. I just considered while doing unnest. But I didn't have much time to think about it because the OSPP was coming to an end. Maybe now I have more time. Thank you for your nice advice and compliment.